h2o-py/h2o/__init__.py

```diff
@@ -61,12 +61,31 @@ def readTxtFromWhl(name, fallback):
 if (__version__.endswith("99999")):
     print(__buildinfo__)
 
-__all__ = ("connect", "init", "api", "connection", "upload_file", "lazy_import", "import_file", "import_sql_table",
+
+__all__ = ["connect", "init", "api", "connection", "upload_file", "lazy_import", "import_file", "import_sql_table",
            "import_sql_select", "parse_setup", "parse_raw", "assign", "deep_copy", "get_model", "get_grid", "get_frame",
            "show_progress", "no_progress", "enable_expr_optimizations", "is_expr_optimizations_enabled", "log_and_echo",
            "remove", "remove_all", "rapids", "ls", "frame", "import_hive_table",
            "frames", "download_pojo", "download_csv", "download_all_logs", "save_model", "download_model", "upload_model", "load_model", "export_file",
            "cluster_status", "cluster_info", "shutdown", "create_frame", "interaction", "as_list", "network_test",
-           "set_timezone", "get_timezone", "list_timezones", "demo", "make_metrics", "cluster", "load_dataset","flow",
+           "set_timezone", "get_timezone", "list_timezones", "demo", "make_metrics", "cluster", "load_dataset", "flow",
            "upload_custom_metric", "upload_custom_distribution",  "mojo_predict_csv", "mojo_predict_pandas", "import_mojo", 
-           "upload_mojo", "print_mojo", "load_grid", "save_grid")
+           "upload_mojo", "print_mojo", "load_grid", "save_grid"]
+
+try:
+    # Export explain functions that are useful for lists of models
+    from h2o.explain import explain, explain_row, variable_importance_heatmap, model_correlation_heatmap, \
+        partial_dependences, register_explain_methods
+
+    __all__ += [
+        "explain",
+        "explain_row",
+        "variable_importance_heatmap",
+        "model_correlation_heatmap",
+        "partial_dependences"
```

Comment:
`partial_dependences` Is it not ok to rename this to `partial_dependence_plots`?

Reply:
@pramitchoudhary I thought the same thing at first... but it's actually just a single plot with multiple lines and not multiple "plots".  The current name, conceptually, makes sense -- my only concern is that people will get confused about whether it's "dependences" or "dependencies".  Also good to note that we have an existing function called `.partial_plot()` which takes a single model and a column (or columns) to generate a PD plot.

Reply:
Got it. Then why not just call it `partial_dependence_plot` so that its clear one is not getting the values but the actual plot and if one is returning the values then we call it `partial_dependence`?

Also, if the `explain` functions are coming from a different namespace then is it not ok to call it `partial_plot()` for consistency?

Reply:
I think using `partial_plot()` could be confusing. (When using jupyter, I often found myself googling the function (just the function name not fully qualified name) instead of waiting for the jupyter's documentation to pop up.)

Reply:
`pdp_plot`?

Reply:
We decided to break this into two functions, one that takes a single model, `h2o.pd_plot()` and one that takes multiple models (as a list or AutoML object), `h2o.pd_multi_plot()`.